spec/services/settings_user_service_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe SettingsUserService do
```

Comment:
the same problem as for spec/services/settings_address_service_spec.rb

Reply:
done, but the comment did not disappear